design-tokens/tokens/layer3/base.json

```diff
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
+  "echoes": {
+    "sizes": {
+      "overlays": {
+        "maxWidth": {
+          "default": {
+            "value": "{echoes.dimension.space.800}",
```

Comment:
@marciopmoreira6 it should reference `size.5000` right?
I'll fix it with a commit, I need to fix the build for layer 3 anyway.

Reply:
thanks